app/src/main/res/layout/fragment_details.xml

```diff
@@ -129,12 +129,11 @@
 
         <ca.antonious.materialdaypicker.MaterialDayPicker
                 android:id="@+id/day_picker"
-                android:layout_width="match_parent"
+                android:layout_width="wrap_content"
```

Comment:
android:layout_width="0dp
android:layout_alignParentStart="true"
android:layout_alignParentEnd="true"

Make the following changes and remove the unnecessary code.